### DISCUSSION TITLE
博客增加音乐播放器应该写在哪个文件？

### DISCUSSION THREAD

**Discussion by fujiwaranatsuho** [MaLuns/hexo-theme-async#125] [Q&A 疑难杂症] [answered]
![image](https://github.com/user-attachments/assets/2df1248a-43f6-4e9d-92b9-2cbb7f426d27)
按照gpt的步骤在footer.js修改了，但是好像找不到main/style.css。我应该在哪个文件写这段代码呢？

**Comment by MaLuns** [comment]
source/_data/style/index.less 可以写在这里

**Reply by fujiwaranatsuho** [reply]
https://blog.csdn.net/weixin_58068682/article/details/116612364 你好，我换成这个教程去导入网易云歌单的播放器，尝试写在\node_modules\hexo-theme-async\layout里面的index.ejs和sidebar文件夹里面的index.ejs、email.ejs，试了这么多地方都没有渲染成功，写在哪个文件才可以正确渲染呢？

**Comment by MaLuns** [comment, marked as answer]
https://stackblitz.com/edit/node-84g1dz

直接使用 aplayer 弄了个示例, 你参考下吧

**Reply by fujiwaranatsuho** [reply]
太感谢了！一个人捣鼓了好几个小时
